src/event/handlers/DiscordMessageLinkHandler.ts

```diff
@@ -0,0 +1,22 @@
+import EventHandler from "../../abstracts/EventHandler";
+import { Message, Constants } from "discord.js";
+import MessagePreviewService from "../../services/MessagePreviewService";
+
+class DiscordMessageLinkHandler extends EventHandler {
+	constructor() {
+		super(Constants.Events.MESSAGE_CREATE);
+	}
+	handle = async (message: Message): Promise<void> => {
```

Comment:
Add a blank line between these two lines, please. Also, `handle` shouldn't be an arrow function as you never access `this`.